doc/classes/CanvasLayer.xml

```diff
@@ -31,6 +38,9 @@
 		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset">
 			The layer's base offset.
 		</member>
+		<member name="own_world" type="bool" setter="set_use_own_world" getter="is_using_own_world">
+			Whether this layer uses its own [code]World2D[/code]; i.e. if this layer has a separate physics space.
```

Comment:
Please document default value too. Docteam usually does it this way: `Default value: [code]false[/code].`